frontend/src/components/Simulator/ContractMethodItem.vue

```diff
@@ -37,8 +38,13 @@ const handleCallReadMethod = async () => {
       }),
     );
 
-    responseMessage.value =
-      typeof result === 'string' ? result : JSON.stringify(result);
+    let repr: string;
+    if (typeof result === 'string') {
```

Comment:
I think type annotations for callReadMethod are incorrect because it specifies that it returns `{ [key: string]: any }`